Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:22.04
 
 # The GS_VERSION argument could be used like this to overwrite the default:
 # docker build --build-arg GS_VERSION=2.21.2 -t geoserver:2.21.2 .
-ARG TOMCAT_VERSION=9.0.65
+ARG TOMCAT_VERSION=9.0.68
```

Comment:
please update the download link below to use https://archive.apache.org/dist/tomcat/tomcat-9/ which provides stable download links otherwise this will be broken again after the next tomcat 9 release

https://github.com/geoserver/docker/blob/422be0ae8d282bb86f50dcbb1005ebb2275d995d/Dockerfile#L56

Reply:
Thanks, will do

Reply:
Done